### ISSUE TITLE
NPE in Build

### ISSUE THREAD

**Issue by Exerosis** [excelsior-oss/excelsior-jet-maven-plugin#66]
Not entirely sure why but I'm getting a null pointer when I build, I might have configured something incorrectly but I'm not getting very much context from the stacktrace:
https://goo.gl/NRpw6U
Full Console: https://goo.gl/g4DE1V

**Comment by pjBooms** [comment]
Thank you for the bug report!
Could you also share Excelsior JET Maven Plugin configuration block to understand what triggered that NPE?

**Comment by Exerosis** [comment]
My apologies here: https://gist.github.com/anonymous/6f31ae91d0a2bb888b7a92c36a79ea06
On a side note, it would be very helpful to support building an exe that runs as admin. As of now my only solutions is packing mt.exe and using it to give the executable a manifest after install. 
Also, I was unable to find welcome screen caption and welcome screen text params for the installer in the maven plugin is there no support for that as of now?
Thank you very much!

**Comment by pjBooms** [comment]
It is duplicate to #62 and fixed in 1.1.2 version of the plugin so please change the version and the problem should be out. 
As to "run as admin", you may add the following configuration to the plugin configuration that will produce the executable that runs as admin:

```xml
<compilerOptions>
  <compilerOption>-runasadmin+</compilerOption>
</compilerOptions>
```
